ypspur_ros:
  ros__parameters:
    base_frame_id: base_link
    odom_frame_id: odom

    right_wheel_joint_id: right_wheel_joint
    left_wheel_joint_id: left_wheel_joint

    control_period: 20
    cmd_vel_timeout: 0.5

    publish_tf: true

    pose_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 10.0]
    twist_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 10.0]

    linear:
      x:
        has_velocity_limitation: true
        v_min: -0.9
        v_max: 0.9
        has_acceleration_limitation: true
        a_min: -1.5
        a_max: 1.5

    angular:
      z:
        has_velocity_limitation: true
        v_min: -3.14
        v_max: 3.14
        has_acceleration_limitation: true
        a_min: -6.28
        a_max: 6.28
